### PR TITLE
Develop

### PR DESCRIPTION
**Pull request by I11ia**
- [DEMO LINK](https://I11ia.github.io/layout_product-cards/)
- [TEST REPORT LINK](https://I11ia.github.io/layout_product-cards/report/html_report/)

### CODE REVIEW

**Review by loralevitska — CHANGES_REQUESTED** [review]
Keep up the good work! Some comments to fix:

File: src/index.html [hunk 1]
```diff
@@ -5,9 +5,47 @@
...
-    <h1>Product cards</h1>
+    <div class="card" data-qa="card">
+      <img
+        class="card__product-image"
+        src="images/imac.jpeg"
+        alt= "APPLE A1419 iMac 27 Retina 5K Monoblock (MNED2UA/A)"
+      >
+
+      <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+      <p class="card__product-code">Product code: 195434</p>
```
Comment (Andriy-Shymkiv) [inline src/index.html:23]:
```suggestion
      <h2 class="card__product-name">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
      
      <p class="card__product-code">Product code: 195434</p>
```

File: src/index.html [hunk 2]
```diff
@@ -5,9 +5,47 @@
...
+
+      <div class="card__review">
+        <div class="stars stars--4">
+          <p class="stars__star"></p>
```
Comment (Andriy-Shymkiv) [inline src/index.html:27]: why paragraph if it is not even a text?

File: src/index.html [hunk 3]
```diff
@@ -5,9 +5,47 @@
...
+        <p class="card__review-amount">Reviews: 5</p>
+      </div>
+
+      <div class="card__price">
+        <p class="card__label-price">Price:</p>
+        <p class="card__actual-price">$2,199</p>
+      </div>
+
+      <a
+        href="#"
```
Comment (Andriy-Shymkiv) [inline src/index.html:43]: please add meaningful href

File: src/styles/blocks/card.scss [hunk 4]
```diff
@@ -0,0 +1,89 @@
...
+    height: 134px;
+  }
+
+  &__product-name {
+    font-weight: 500;
+    font-size: 12px;
+    line-height: 18px;
+    color: $primary-color;
+    text-align: left;
+    margin: 40px 0 0;
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/card.scss:22]: you can just reset margin for all `p` in body and here just `margin-top`, the same everywhere

File: src/styles/blocks/default-styles.scss [hunk 5]
```diff
@@ -0,0 +1,9 @@
+* {
+  margin: 0;
+  padding: 0;
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/default-styles.scss:3]: not a good practice to reset indent for every element in page, just do it for those you really need

File: src/styles/blocks/stars.scss [hunk 6]
```diff
@@ -0,0 +1,23 @@
...
+    &:last-child {
+      margin-right: 0;
+    }
+  }
+
+  @for $i from 1 through 5 {
+    &--#{$i} :nth-of-type(-n + #{$i}) {
+      background-image: url("../images/star-active.svg");
+    }
+  }
```
Comment (Andriy-Shymkiv) [inline src/styles/blocks/stars.scss:22]: 🔥

**Review by Andriy-Shymkiv — CHANGES_REQUESTED** [review]

File: src/styles/blocks/default-styles.scss [hunk 7]
```diff
@@ -0,0 +1,14 @@
+* {
+  // margin: 0;
+  // padding: 0;
```
Comment (Viktor-Kost) [inline src/styles/blocks/default-styles.scss:3]: remove comments

File: src/styles/blocks/default-styles.scss [hunk 8]
```diff
@@ -0,0 +1,14 @@
...
+  box-sizing: border-box;
+  font-family: Roboto, Halvetica, sans-serif;
+  font-weight: 400;
+  font-size: 10px;
+  line-height: 14px;
```
Comment (Viktor-Kost) [inline src/styles/blocks/default-styles.scss:8]: it's only allowed to use box-sizing : border-box at * selector. So move other rules to html/body tag selectors

File: src/styles/blocks/card.scss [hunk 9]
```diff
@@ -0,0 +1,87 @@
...
+    align-items: center;
+    text-transform: uppercase;
+    text-decoration: none;
+    font-size: 14px;
+    line-height: 16px;
+    font-weight: 700;
+    margin-top: 16px;
+    color: $secondary-button-buy-color;
+    background-color: $primary-button-buy-color;
+    height: 40px;
```
Comment (Viktor-Kost) [inline src/styles/blocks/card.scss:76]: it's not a good practice to hardcode height for elements with content inside. Use padding instead

File: src/styles/blocks/stars.scss [hunk 10]
```diff
@@ -0,0 +1,23 @@
...
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin: 0 4px 0 0;
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position-x: center;
+    background-position-y: center;
```
Comment (Viktor-Kost) [inline src/styles/blocks/stars.scss:11]: apply as background `shorthand`

File: src/styles/blocks/stars.scss [hunk 11]
```diff
@@ -0,0 +1,23 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    margin: 0 4px 0 0;
```
Comment (Viktor-Kost) [inline src/styles/blocks/stars.scss:7]:
```suggestion
```

File: src/styles/blocks/stars.scss [hunk 12]
```diff
@@ -0,0 +1,23 @@
...
+    background-image: url("../images/star.svg");
+    background-repeat: no-repeat;
+    background-position-x: center;
+    background-position-y: center;
+
+    &:last-child {
+      margin-right: 0;
+    }
```
Comment (Viktor-Kost) [inline src/styles/blocks/stars.scss:15]:
```suggestion
    &:not(:last-child) {
      margin-right: 4px;
    }
```

File: src/styles/blocks/card.scss [hunk 13]
```diff
@@ -0,0 +1,87 @@
...
+  align-items: center;
+  flex-direction: column;
+  border: 1px solid #f3f3f3;
+  border-radius: 1px;
+  width: 200px;
+  padding: 32px 16px 16px;
+
+  &__product-image {
+    width: 160px;
+    margin: 0 3px;
```
Comment (Viktor-Kost) [inline src/styles/blocks/card.scss:12]:
```suggestion
    margin: 0 auto;
```

**Review by Viktor-Kost — CHANGES_REQUESTED** [review]
GJ!
Some minor fixes needed

**Review by mykhalenych — APPROVED** [review]

**Pull request closed without merging**